### ISSUE TITLE
Spawning Numerous WMIC Processes

### ISSUE THREAD

**Issue by AROAH1337** [Vencord/Vesktop#428]
**Describe the bug**

Vesktop is spawning countless WMIC processes that are for some reason attempting to scan all running processes and save the results as a CSV file. The exact command line is 'C:\Windows\system32\cmd.exe /d /s /c "wmic process get ProcessID,ExecutablePath /format:csv"' Upon killing enough of the rampant processes, I received an update notification, so I'm assuming it's something awry in the update check procedure. 

**To Reproduce**

Starting Vencord and letting it run momentarily.

**Expected behavior**

Not... doing that.

**Screenshots**

![image](https://github.com/Vencord/Vesktop/assets/7283348/748032fe-9b3a-45de-b060-fd29fd4685f2)
![image](https://github.com/Vencord/Vesktop/assets/7283348/98c99109-7ca5-41ec-ba83-7123dbceaf46)

**Desktop (please complete the following information):**
 - OS/Distro: Windows 11 22H2
 - Version: 22621.3155

**Command line output**

```
PS C:\Users\AROAH> .\Documents\Apps\Vesktop\Vesktop.exe
PS C:\Users\AROAH>
[arRPC > ipc] listening at \\?\pipe\discord-ipc-0
[arRPC > websocket] listening on 6463
[arRPC > process] started
 [INFO] [DSH]  vesktop.exe
 [INFO] [DSH]  Creating WndMsg Listener Window
 [INFO] [DSH]  Get number of capabilities
 [INFO] [DSH]  Get stream caps: 0
 [INFO] [DSH]  Get stream caps: 1
 [INFO] [DSH]  Get stream caps: 2
 [INFO] [DSH]  Get stream caps: 3
 [INFO] [DSH]  Get stream caps: 4
 [INFO] [DSH]  Get stream caps: 5
 [INFO] [DSH]  Get stream caps: 6
 [INFO] [DSH]  Get stream caps: 7
 [INFO] [DSH]  Get stream caps: 8
 [INFO] [DSH]  Get stream caps: 9
 [INFO] [DSH]  Get stream caps: 10
 [INFO] [DSH]  Get stream caps: 11
 [INFO] [DSH]  Get stream caps: 12
 [INFO] [DSH]  Get stream caps: 13
 [INFO] [DSH]  Get stream caps: 14
 [INFO] [DSH]  Get stream caps: 15
 [INFO] [DSH]  Get stream caps: 16
 [INFO] [DSH]  Get stream caps: 17
 [INFO] [DSH]  Get stream caps: 18
 [INFO] [DSH]  Get stream caps: 19
 [INFO] [DSH]  Get stream caps: 20
 [INFO] [DSH]  Get stream caps: 21
 [INFO] [DSH]  Get stream caps: 22
 [INFO] [DSH]  Get stream caps: 23
 [INFO] [DSH]  Get stream caps: 24
 [INFO] [DSH]  Get stream caps: 25
 [INFO] [DSH]  Get stream caps: 26
 [INFO] [DSH]  Get stream caps: 27
 [INFO] [DSH]  Get stream caps: 28
 [INFO] [DSH]  Get stream caps: 29
 [INFO] [DSH]  Get stream caps: 30
 [INFO] [DSH]  Get stream caps: 31
 [INFO] [DSH]  Get stream caps: 32
 [INFO] [DSH]  Destroying parent object
 [INFO] [DSH]  Destroying WndMsg Listener Window
 [INFO] [DSH]  Destroyed window
 [INFO] [DSH]  Unregistered window class
```

**Additional context**

This wasn't a problem (as far as I noticed) until fairly recently. In fact, I only noticed because WmiPrvSE started using 100% of my CPU and started looking into tracing what was causing it. Vencord itself was otherwise working as expected.

**Comment by Vendicated** [comment]
it's arrpc. they shouldn't stay alive though, they should promptly exit again though, this is very strange behaviour. can you still reproduce after restarting?

**Comment by AROAH1337** [comment]
I didn't notice that the "Failed to connect to arRPC" message wasn't showing up. It's acting fine after restarting Windows. 

I also found a number of event logs under WMI-Activity that would indicate that these WMIC queries were running for 20 minutes in an idle state ("PossibleCause = Throttling Idle Tasks, refer to CIMOM regkey: ArbTaskMaxIdle"), so I wonder if some weird problem was holding up the query, but the arRPC component assumed that the query failed, so it ran another one? I'm getting into weeds I'm not familiar with on this one.

I guess I got ahead of myself and should have just tried restarting before I started digging. 😅
